assets/stylesheets/host.css

```diff
@@ -9,4 +10,231 @@ body {
 .frame {
   height: 100%;
   width:  100%;
+  margin-top: 15px;
+}
+
+.modal-container {
+  animation: modalContainerFade ease 0.5s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: none;
+  z-index: 1;
+}
+
+@keyframes modalContainerFade {
+  0% {
+    background-color: rgba(0, 0, 0, 0);
+  }
+  100% {
+    background-color: rgba(0, 0, 0, 0.5);
+   }
+}
+
+.modal {
+  animation: modalFade ease 0.5s;
+  background-color: white;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 15px;
+  width: 1000px;
+  height: 550px;
+  border: 5px solid;
+  border-radius: 20px;
+}
+
+@keyframes modalFade {
+  0% {
+   opacity: 0%;
+  }
+  100% {
+    opacity: 100%;
+   }
+}
+
+.modal-text {
+  width: 950px;
+  height: 450px;
+  margin: 25px 25px 0px 25px;
+  overflow: scroll;
+}
+
+.modal-activity-title {
+  margin-top: 0px;
```

Comment:
`0px` and `0vw` and `0%` can all just be written as `0`.

Reply:
Noted & done.